freqtrade/optimize/__init__.py

```diff
@@ -16,9 +16,16 @@
 from freqtrade import misc, constants, OperationalException
 from freqtrade.exchange import Exchange
 from freqtrade.arguments import TimeRange
+import importlib
+ujson_found = importlib.util.find_spec("ujson")
+if ujson_found is not None:
+    import ujson
```

Comment:
What's the reason to import ujson here again? 
it's imported in line 5 ... and falls back to json if it doesn't find it ...

Reply:
it was in the bslap branch. I admit I didn't understand it either. probably should be removed

Reply:
I think its a remnant when bslap was being toyed with for backtest`esque use. Berlin had it in his backtest rig checking many thousands of scenarios with a lot of json input. 

Its not that appropriate here. 
im using influxdb for candle servers, so no json at all, and in this use-case / pr the gain is minimal